cloudify_cli/commands/audit_log.py

```diff
@@ -0,0 +1,51 @@
+import click
+
+from ..cli import cfy, helptexts
+from ..table import print_data
+
+AUDITLOG_COLUMNS = ['ref_table', 'ref_id', 'operation', 'creator_name',
+                    'execution_id', 'created_at']
+
+
+@cfy.group(name='auditlogs')
+@cfy.assert_manager_active()
+def auditlogs():
+    """Manage the audit log"""
+    pass
+
+
+@auditlogs.command(name='list',
+                   short_help='List audit log entries')
+@click.option('-c', '--creator-name',
+              help=helptexts.AUDIT_CREATOR_NAME)
+@click.option('-e', '--execution-id',
+              help=helptexts.AUDIT_EXECUTION_ID)
+@cfy.options.sort_by()
+@cfy.options.descending
+@cfy.options.pagination_offset
+@cfy.options.pagination_size
+@cfy.options.common_options
+@cfy.pass_logger
+@cfy.pass_client()
+def get(creator_name,
```

Comment:
Shouldn't this be list rather than get?

Reply:
Here are my reasons:
* I believe we won't ever need (I already know how it sounds…) to get a single audit log entry,
* I don't like to shadow `list()` function (or any other for that matter) unless I absolutely have to,
* it's a REST's GET after.

Reply:
how about list_logs, then

Reply:
Seconding @tehasdf 's suggestion- whatever we do let's not call this get, since in the CLI that has semantics of "One thing".